cli/create_test.go

```diff
@@ -864,24 +868,55 @@ func TestCreateValidateRichParameters(t *testing.T) {
 		coderdtest.AwaitTemplateVersionJobCompleted(t, client, version.ID)
 		template := coderdtest.CreateTemplate(t, client, owner.OrganizationID, version.ID)
 
-		inv, root := clitest.New(t, "create", "my-workspace", "--template", template.Name)
-		clitest.SetupConfig(t, member, root)
-		pty := ptytest.New(t).Attach(inv)
-		clitest.Start(t, inv)
+		t.Run("Prompt", func(t *testing.T) {
+			inv, root := clitest.New(t, "create", "my-workspace-1", "--template", template.Name)
+			clitest.SetupConfig(t, member, root)
+			pty := ptytest.New(t).Attach(inv)
+			clitest.Start(t, inv)
 
-		matches := []string{
-			listOfStringsParameterName, "",
-			"aaa, bbb, ccc", "",
-			"Confirm create?", "yes",
-		}
-		for i := 0; i < len(matches); i += 2 {
-			match := matches[i]
-			value := matches[i+1]
-			pty.ExpectMatch(match)
-			if value != "" {
-				pty.WriteLine(value)
+			matches := []string{
+				listOfStringsParameterName, "",
+				"aaa, bbb, ccc", "",
+				"Confirm create?", "yes",
 			}
-		}
+			for i := 0; i < len(matches); i += 2 {
+				match := matches[i]
```

Comment:
In [this PR](https://github.com/coder/coder/pull/15406) Spike replaced the for-loop with a ladder of matches. IMHO it is a more reader-friendly form.

Reply:
Done! 🪜